stories/Introduction/Cheatsheet/componentsFamilies/SelectionFamily.js

```diff
@@ -421,4 +407,62 @@ class StarsRatingBarExample extends PureComponent {
   }
 }
 
+class FacesRatingBarExample extends PureComponent {
+  state = { value: 2 };
+
+  onRatingChange = value => this.setState({ value });
+
+  render() {
+    const { value } = this.state;
+
+    const symbol = selectionSymbols.facesRatingBar;
+    const components = selectionSymbolsToComponents[symbol];
+
+    const singleComponentProps = {
+      name: createLinkedSymbolName({ groupSymbol: Category.SELECTION, symbol }),
```

Comment:
both "Faces Rating Bar" & "Stars Rating Bar" have no UX story. Just a component story. Therefore, it should be changed to :
```suggestion
      name: symbol,
```